tests/dataset_runner.py

```diff
@@ -228,6 +230,20 @@ def wait_for_primary_bundles(self):
             raise RuntimeError(f'Expected {self.expected_bundle_count} primary bundles, but only '
                                f'got {primary_bundles_count}')
 
+    def assert_valid_schema_versions_in_provenance(self):
```

Comment:
Could we please move the assert function down to join the group of other assert functions below? (Roughly line 387).

Reply:
Perhaps also write a small docstring for this function please so that folks who aren't familiar with the schema versions in the provenance can understand what is being validated here?